CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Changed
 
 - Use custom policy to generate cloudfront signed url
+- Upgrade node to version 16, the current LTS
```

Comment:
You should specify that aws is not concerned by this upgrade.

`Upgrade node to version 16, , the current LTS, for front and mail apps` ?